src/transaction.js

```diff
@@ -702,6 +713,13 @@ Transaction.prototype.__toBuffer = function (buffer, initialOffset, __allowWitne
   if (this.version >= 3 && this.zcash) {
     writeInt32(this.version | (1 << 31))
     writeUInt32(this.versionGroupId)
+    if (this.version === 4) {
+      // THIS DOES NOT SUPPORT SAPLING GENERALLY - THESE ARE DUMMIES THAT ASSUME NO SHIELDED SPENDS OR OUTPUTS
```

Comment:
Hmm, this concerns me a bit. If I understand this correctly, that means the pool won't be able to add shielded utxos to blocks it generates?